utilities/test_suite/rpp_test_suite_common.h

```diff
@@ -849,7 +850,7 @@ inline void read_image_batch_turbojpeg(Rpp8u *input, RpptDescPtr descPtr, vector
         fseek(fp, 0, SEEK_END);
         long jpegSize = ftell(fp);
         rewind(fp);
-        unsigned char* jpegBuf = (unsigned char*)malloc(jpegSize);
+        unsigned char* jpegBuf = (unsigned char*)calloc(jpegSize, sizeof(Rpp8u));
```

Comment:
What is this change for?

Reply:
We are having callocs every where in the test suite for initializing buffers except for this case and also It will store zeros by default to the buffer